pom.xml

```diff
@@ -429,7 +429,12 @@
       <dependency>
         <groupId>org.apache.commons</groupId>
         <artifactId>commons-lang3</artifactId>
-        <version>3.12.0</version>
+        <version>3.13.0</version>
+      </dependency>
+      <dependency>
```

Comment:
Please use guava instead, as discussed.

Reply:
commons is used in several places other than this current change.